packages/wonder-blocks-link/components/link.js

```diff
@@ -131,6 +144,9 @@ export default class Link extends React.Component<SharedProps> {
             this.context.router,
         );
 
+        // `"_self"` is the default `target` on anchor tags.
+        const target = openInNewTab ? "_blank" : "_self";
```

Comment:
TIL.  This change will result in lots of the snapshots changing.  I wonder if we should maybe do:
```
const target = openInNewTab ? "_blank" : undefined;
```

Reply:
If that works!

Reply:
I'm going to drop this all together given your suggestion of just letting `LinkCore` handle things.